greenberry/greenberry_lex_test.py

```diff
@@ -0,0 +1,76 @@
+from gb_utils.greenberry_lex import GreenBerryLex
+from symbols import *
+from debug_cp import *
+import inspect
+MATH_OPS = ["+", "-", "*", "/"]
+BOOLS = [S.TRUE, S.FALSE]
+BOOL_OPS = [S.GREATER, S.LESS]
+EOS = [S.NL, S.EOF]
+
+
+def greenberry_lex_test(x, expected):
+    KWDs = [
+        getattr(S, i)
+        for i in [
+            b[0]
+            for b in [
+                a
+                for a in inspect.getmembers(S, lambda a: not inspect.isroutine(a))
+                if not (a[0].startswith("__") and a[0].endswith("__"))
+            ]
+        ]
+    ]
```

Comment:
Though a work of art, those comprehensions will give me nightmares while reading the source again. Could you use normal loops? Thank You!

Reply:
@thisusernameisavailable